commands-yml/commands/element/other/active.yml

```diff
@@ -0,0 +1,74 @@
+---
+name: Get Active Element
+short_description: Gets the active element of the current session
+description:
+  >
+    Get the element on the page that currently has focus.
```

Comment:
This doesn't seem to add much.

Reply:
Yeah, that was before when description was mandatory. I'll take it out.